test/test-gotest.sh

```diff
@@ -19,6 +19,12 @@ if [[ "$@" = *"--root"* ]]; then
 	XTAGS='-tags root'
 fi
 
+# As per https://github.com/travis-ci/docs-travis-ci-com/blob/master/user/docker.md
+# Docker is not supported on Travis macOS test instances.
+if [[ "$TRAVIS_OS_NAME" == "osx" ]];then
+	XTAGS="-tags nodocker $XTAGS"
```

Comment:
This breaks if the conditional above is true because only one `-tags` argument may be specified. I'll fix this.